app/controllers/match.js

```diff
@@ -0,0 +1,52 @@
+// require list
+//const slug = require("slug");
+//const mongoose = require("mongoose");
+
+// display the user people that can be matched with
+// items needed from database: User image, name, city
+// for each person in database
+
+function showUser() {
+    //total users in database
+    const users = 10; //users.length
+    const userDetails = [];
+    let i = 0;
+
+    for (i = 0; i <= users; i++) {

```

Comment:
`let i` should be defined in the for loop
```suggestion
    for (let i = 0; i <= users; i++) {
```